CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2108,8 +2119,44 @@ private void SetPopupSizeAndPositionImpl(Rect rect)
             popupImage.Width = rect.Width;
             popupImage.Height = rect.Height;
 
-            Canvas.SetLeft(popupImage, rect.X);
-            Canvas.SetTop(popupImage, rect.Y);
+            int x = rect.X,
```

Comment:
This whole block of logic would be better in the actual translation class.

Then it can be easily unit tested.

Reply:
Moved it to the class.